docs/connectors/getting-started-connectors/quickstart.yml

```diff
@@ -16,7 +16,8 @@ spec:
   description: !snippet README.adoc#description
   prerequisites:
     - A Red Hat identity
-    - A running Kafka instance (see the Getting started with Kafka quick start)
+    - You've created a  {product-kafka} instance and the instance is in *Ready* state. See the _Getting started with OpenShift Streams for Apache Kafka_ quick start.
```

Comment:
I'm not sure if you can use an adoc attribute here. Regardless, I think the correct term would simply be "Kafka instance".

Reply:
yes - I was trying to be specific -it must be an OpenShift Streams Kafka instance - not just any Kafka instance